tests/js_api/src/api.ts

```diff
@@ -7,6 +7,8 @@ export function createApi(): Promise<ApiPromise> {
     // Create the API and wait until ready
     return ApiPromise.create({
```

Comment:
@kroos47 Do you think we should use our wrapper here as well? This is ts and that is in js though.